parser/sql/dialect/oracle/src/main/antlr4/imports/oracle/PLSQL.g4

```diff
@@ -279,10 +283,51 @@ autonomousTransPragma
     : PRAGMA AUTONOMOUS_TRANSACTION SEMI_
     ;
 
-dmlEventClause
-    : dmlEventElement (OR dmlEventElement)* ON viewName
+plsqlTriggerSource
+    : (schemaName DOT_)? triggerName sharingClause? defaultCollationClause? (simpleDmlTrigger | systemTrigger)
     ;
 
-dmlEventElement
-    : (DELETE | INSERT | UPDATE) (OF LP_ columnName (COMMA_ columnName)* RP_)?
+simpleDmlTrigger
+    : (BEFORE | AFTER) dmlEventClause
+    ;
+
+dmlEventClause
+    : dmlEventTrigger (OR dmlEventTrigger)* ON viewName
+    ;
+
+dmlEventTrigger
+    : DELETE
```

Comment:
Please keep these branches in one line.
Refer code conduct - https://shardingsphere.apache.org/community/en/involved/conduct/code/.
> If a rule’s branch is over than 5, every branch take a new line.

Reply:
Done. Thanks